src/RdfFieldHandlerInterface.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\rdf_entity;
+
+/**
+ * Provides a contract for the RDF entity field handler service.
+ */
+interface RdfFieldHandlerInterface {
```

Comment:
If we are creating an interface, we should replace instances of injection of the service so that  the other services can use an interface instead. I see the `QueryFacotry`, the `SparqlCondition`, the `RdfEntitySparqlStorage` and the `Query` classes are still taking a `RdfFieldHanlder` as a parameter in their constructor.

Reply:
@idimopoulos, I think this was already fulfilled in your commit. So, this could be merged, no?

Reply:
yy I committed it already. We can proceed with merging.

Reply:
@idimopoulos, your commit looks good. Please merge.